vignettes/smatr.Rmd

```diff
@@ -15,5 +15,176 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
+#Load library
```

Comment:
For a vignette, we want to separate the code below into several chunks, adding some of the text in comments as text in the document, and removing the results. The results will be generated when the vignette is built.

Reply:
Added more chunks and text, mostly copy and pastes from help files and paper 3492bcbe13b827450a2d1400e54ff2ea236f4b27  I think any detailed theory or specifics I will have to leave to creators!